doc/ref_model/chapters/appendix-a.md

```diff
@@ -50,9 +106,9 @@ Current CNTT Policies:
 
 **SR-IOV:**
 
-CNTT recognizes that today, SR-IOV provides a critical Capability for increased throughput over network interfaces at an economical cost. As such, the CNTT approach to SR-IOV is to (detailed policy language under development; to be tied in with VNF Evolution).
+CNTT recognizes that today, SR-IOV provides a critical Capability for increased throughput over network interfaces at an economical cost. As such, the CNTT approach to SR-IOV is to accept the use of SR-IOV, and encourage the development of a suitable replacement that address the management and deployment shortcomings of SR-IOV while achieving similar price and performance.  When such an alternative emerges, this SR-IOV exception will be revisited.  (detailed policy language under development; to be tied in with VNF Evolution).
```

Comment:
I'm sorry to keep commenting on this section, but I think my previous suggestions decreased the readability of this section rather than improving it.  I'd like to propose another change (in effect, re-arranging the text of these two paragraphs.  
```suggestion
```